manifests/init.pp

```diff
@@ -27,10 +30,16 @@
   $valid_ensures = [ 'absent', 'present', 'latest' ]
   validate_re($ensure, $valid_ensures)
 
+  file { $name:
```

Comment:
This will be a duplicate resource error when `ensure=absent`, due to the code block below.